resources/views/ItemList/Components/Filter/ItemFilterList.twig

```diff
@@ -1,75 +0,0 @@
-{# Include storefront template translations #}
-{% import "Ceres::PageDesign.Macros.Translations" as Translations %}
-{{ Translations.add( "Findologic", "Template" ) }}
```

Comment:
I currently do not see any translations, this will still need to be loaded somewhere:

![image](https://user-images.githubusercontent.com/51320851/213695944-7cbe93bc-912c-464b-96f1-21ac8fb2d782.png)